.circleci/config.yml

```diff
@@ -398,7 +411,7 @@ jobs:
             ccache -sz -M 5Gi
             source /opt/rh/gcc-toolset-9/enable
             make clean
-            make debug EXTRA_CMAKE_FLAGS="-DVELOX_BUILD_TESTING=OFF -DVELOX_ENABLE_SUBSTRAIT=ON" NUM_THREADS=16 MAX_HIGH_MEM_JOBS=8 MAX_LINK_JOBS=16
+            make release EXTRA_CMAKE_FLAGS="-DVELOX_ENABLE_SUBSTRAIT=ON" NUM_THREADS=16 MAX_HIGH_MEM_JOBS=8 MAX_LINK_JOBS=16
```

Comment:
@kgpai Is Substrait support required to be part of the minimal build?

Reply:
> @kgpai Is Substrait support required to be part of the minimal build?

Hi, @majetideepak  Substrait support is not required to be part of the minimal build.